wandb/sdk/artifacts/artifact.py

```diff
@@ -638,6 +651,45 @@ def updated_at(self) -> str:
         assert self._created_at is not None
         return self._updated_at or self._created_at
 
+    @property
+    def ttl(self) -> Optional[datetime.timedelta]:
+        """Artifact TTL(time to live).
+
+        Once the artifact's created_at is older than the ttl, the artifact is marked as deleted.
+        To turn off previously set artifact TTL, set TTL to wandb.ArtifactTTL.DELETE
+        EX: Artifact.created_at = 1/1/2000, ttl=10 days.
+            This artifact will be marked for deletion on 1/11/2000.
+        """
+        if (
+            self._ttl_duration_seconds is None
+            or self._ttl_duration_seconds == ArtifactTTL.DELETE.value
```

Comment:
How is `self._ttl_duration_seconds == ArtifactTTL.DELETE.value` different than `self._ttl_duration_seconds is None`? In both cases we're saying there's no TTL (and the artifact will inherit the default from its collection), no? Also, it's confusing that:
```
artifact.ttl(ArtifactTTL.DELETE)
ttl = artifact.ttl  # I would expect this to be `ArtifactTTL.DELETE`, but it's `None`
```

Reply:
Yes great point.
Changing ArtifactTTL.DELETE -> ArtifactTTL.DEFAULT (does the same thing as before but will be better for explaining inheritance in the next iteration of TTL)